content/en/monitors/notify/_index.md

```diff
@@ -116,6 +117,16 @@ Disk space is low @ops-team@company.com
 
 {{% notifications-integrations %}}
 
+### Workflows
+
+[Datadog Workflow Automation][9] allows you to orchestrate and automate your end-to-end processes by building workflows made up of actions that connect to your infrastructure and tools. You can add a pre-configured flow from an out-of-the-box blueprint, add an existing custom workflow or create a new workflow. 
+
+In the message section, add the full workflow mention name:
+   - The mention name should start with `@workflow-`. For example, `@workflow-my-workflow`
+   - To pass trigger variables into the workflow, use a comma-separated list with the syntax `@workflow-name(key=value, key=value)`. For example, `@workflow-my-workflow(name="Bits", alert_threshold=threshold)`
+
+You can also click **Add Workflow** below the message section and select from existing workflows. To create a workflow, click the **+** icon. For more information on how to trigger a workflow, see the [Workflow Automation][10] docs.
```

Comment:
```suggestion
You can trigger a [workflow automation][9] or create a new workflow from a monitor.

**To add an existing workflow to a monitor**:
1. In the message section, add the full workflow mention name:
   - The mention name should start with `@workflow-`. For example, `@workflow-my-workflow`
   - To pass trigger variables into the workflow, use a comma-separated list with the syntax `@workflow-name(key=value, key=value)`. You can use message template variables as trigger variables. For example, `@workflow-my-workflow(hostname={{host.name}})`

1. Alternatively, click **Add Workflow** and search for it in the drop-down menu.

**To create a workflow**:
1. Click **Add Worklfow**. 
1. Click the **+** icon and select a Blueprint, or select **Start From Scratch**. 
   {{< img src="/monitors/create-workflow.png" alt="Click the + button to add a new workflow" style="width:90%;">}}

For more information on how to trigger a workflow, see the [Workflow Automation][10] docs.
```
- Let's stick to business here and get the user on their way. We link off for more info.
- I couldn't figure out where the **+** button was at first. I think we might need a screenshot here. I'll add it after I finish my review